src/layouts/Nav/nav.module.styl

```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
   padding-left $navbar-padding-vertical
   padding-right $navbar-padding-vertical
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
+
+
+.hamburger
+  flex 1
+  align-self center
+  @media (min-width: 760px)
```

Comment:
This should use the `$above` function and the breakpoints set in the scale function

above func https://github.com/realworldreact/jamcommerce/blob/master/src/functions.styl#L142

scale https://github.com/realworldreact/jamcommerce/blob/master/src/vars.styl#L67